pkg/controller/dgraph/models/query/login.go

```diff
@@ -80,7 +80,7 @@ func CheckLogin(username, inputPassword string) bool {
 		logrus.Error(err)
 		return false
 	}
-	return comparePasswords(login.Password, []byte(inputPassword))
+	return username == "admin" && comparePasswords(login.Password, []byte(inputPassword))
```

Comment:
instead of this the ideal way is to update the query in `GetHashedPassword(username)` to check for username match using `@filter(eq(username, usernameFromFunctionArgument))`